pcdsdevices/epics_motor.py

```diff
@@ -159,6 +204,23 @@ def check_value(self, value):
             raise MotorDisabledError("Motor is not enabled. Motion requests "
                                      "ignored")
 
+    def check_limit_switches(self):
```

Comment:
This function is missing the case where both limits are hit. This is nonphysical but if the IOC reports this we should know.